### ISSUE TITLE
ISBAT CoBlocks and Gutenberg conflicting - Locking the Edit Post page

### ISSUE THREAD

**Issue by jonesch** [godaddy-wordpress/coblocks#1772]
### Describe the bug:
We have a few sites that are running both CoBlocks Version 2.5.3 and left Gutenberg Version 9.5.0. With thing with the latest release of Gutenberg, we are occasionally getting the editor page to lock up. Thus making the site uneditable. 

### To reproduce:
Activate both versions of the plugins mentioned above and the following gets logged to the console in Chrome Dev Tools
```
react-dom.min.de439aae.js:125 TypeError: e is not a function
    at index.js?ver=d72cf0f…c1d0889ea77f62252:1
    at Array.forEach (<anonymous>)
    at index.js?ver=d72cf0f…c1d0889ea77f62252:1
    at react-dom.min.de439aae.js:128
    at unstable_runWithPriority (react.min.e713ea3b.js:25)
    at Da (react-dom.min.de439aae.js:60)
    at Ch (react-dom.min.de439aae.js:128)
    at Dh (react-dom.min.de439aae.js:133)
    at Aj (react-dom.min.de439aae.js:158)
    at unstable_runWithPriority (react.min.e713ea3b.js:25)
Me @ react-dom.min.de439aae.js:125
Jh.f.componentDidCatch.c.callback @ react-dom.min.de439aae.js:139
Wg @ react-dom.min.de439aae.js:67
oj @ react-dom.min.de439aae.js:127
Aj @ react-dom.min.de439aae.js:160
unstable_runWithPriority @ react.min.e713ea3b.js:25
Da @ react-dom.min.de439aae.js:60
ab @ react-dom.min.de439aae.js:154
Te @ react-dom.min.de439aae.js:146
(anonymous) @ react-dom.min.de439aae.js:61
unstable_runWithPriority @ react.min.e713ea3b.js:25
Da @ react-dom.min.de439aae.js:60
Pg @ react-dom.min.de439aae.js:61
ha @ react-dom.min.de439aae.js:60
mk.Events.current @ react-dom.min.de439aae.js:236
Ei @ react-dom.min.de439aae.js:41
```

If either plugin is disabled, then there are no console errors and users are able to edit their post content.

### Expected behavior:
We expect to be able to fully edit the post content

### Screenshots:
![image](https://user-images.githubusercontent.com/1905918/101420544-71db3600-38c0-11eb-85b6-4e54350bfe8b.png)

### Isolating the problem:

- [ ] This bug happens with no other plugins activated
- [ ] This bug happens with a default WordPress theme active
- [ ] This bug happens **without** the Gutenberg plugin active
- [x] I can reproduce this bug consistently using the steps above

### WordPress version:
v5.5.3

### Gutenberg version:
v9.5.0

Thanks!!

**Comment by sirreal** [comment]
I've also observed this with CoBlocks and Gutenberg 9.5.0 and 9.5.1.

It's easy to reproduce by adding a paragraph block and then unselecting it.

**Comment by sirreal** [comment]
This appears to be from a change in behavior in Gutenberg and was addressed in https://github.com/WordPress/gutenberg/pull/27634.

The issue is with Gutenberg versions 9.5.0 and 9.5.1. The issue was not present before 9.5.0 and is fixed in 9.5.2.

The issue was that the `subscribe` method does not return an `unsubscribe` callback. This would be a nice change to make to the store although it is [not strictly required according to existing documentation](https://github.com/WordPress/gutenberg/blob/77be77c60fc8bedc9c930d3c89a783a1b0f4be2f/packages/data/README.md#generic-stores).

https://github.com/godaddy-wordpress/coblocks/blob/dee8e8c9da7804de6d6347c27290e3a6cda3b43b/src/extensions/coblocks-settings/coblocks-settings-store.js#L123-L125

I believe this can be closed.

**Comment by jonesch** [comment]
We went back to the site(s) that gave us the issue and sure enough, they are all fixed with the 9.5.2!!
Thank you @sirreal for following up here 🙇‍♂️
